common/db_test.go

```diff
@@ -578,3 +582,311 @@ func TestDBUnbondingLocks(t *testing.T) {
+	}
+
+	row := dbraw.QueryRow("SELECT sender, recipient, faceValue, winProb, senderNonce, recipientRand, recipientRandHash, sig, sessionID FROM winningTickets")
+	var actualSender, actualRecipient, actualRecipientRandHash, actualSessionID string
+	var actualFaceValueBytes, actualWinProbBytes, actualRecipientRandBytes, actualSig []byte
+	var actualSenderNonce uint32
+	err = row.Scan(&actualSender, &actualRecipient, &actualFaceValueBytes, &actualWinProbBytes, &actualSenderNonce, &actualRecipientRandBytes, &actualRecipientRandHash, &actualSig, &actualSessionID)
+
+	assert := assert.New(t)
+	assert.Equal(ticket.Sender.Hex(), actualSender)
+	assert.Equal(ticket.Recipient.Hex(), actualRecipient)
+	assert.Equal(ticket.FaceValue, new(big.Int).SetBytes(actualFaceValueBytes))
+	assert.Equal(ticket.WinProb, new(big.Int).SetBytes(actualWinProbBytes))
+	assert.Equal(ticket.SenderNonce, actualSenderNonce)
+	assert.Equal(recipientRand, new(big.Int).SetBytes(actualRecipientRandBytes))
+	assert.Equal(ticket.RecipientRandHash, ethcommon.HexToHash(actualRecipientRandHash))
+	assert.Equal(sig, actualSig)
+	assert.Equal(sessionID, actualSessionID)
+
+	ticketsCount := getRowCountOrFatal("SELECT count(*) FROM winningTickets", dbraw, t)
+	assert.Equal(1, ticketsCount)
+}
+
+func TestStoreWinningTicket_GivenNilTicket_ReturnsError(t *testing.T) {
+	dbh, dbraw, err := TempDB(t)
+	defer dbh.Close()
+	defer dbraw.Close()
+	if err != nil {
+		t.Error(err)
+		return
+	}
+	sig := randBytesOrFatal(42, t)
+	recipientRand := new(big.Int).SetInt64(1234)
+
+	err = dbh.StoreWinningTicket("sessionID", nil, sig, recipientRand)
+
+	assert := assert.New(t)
+	assert.NotNil(err)
+	assert.Contains(err.Error(), "nil ticket")
+}
+
+func TestStoreWinningTicket_GivenNilSig_ReturnsError(t *testing.T) {
+	dbh, dbraw, err := TempDB(t)
+	defer dbh.Close()
+	defer dbraw.Close()
+	if err != nil {
+		t.Error(err)
+		return
+	}
+	sessionID, ticket, _, recipientRand := defaultWinningTicket(t)
+
+	err = dbh.StoreWinningTicket(sessionID, ticket, nil, recipientRand)
+
+	assert := assert.New(t)
+	assert.NotNil(err)
+	assert.Contains(err.Error(), "nil sig")
+}
+
+func TestStoreWinningTicket_GivenNilRecipientRand_ReturnsError(t *testing.T) {
+	dbh, dbraw, err := TempDB(t)
+	defer dbh.Close()
+	defer dbraw.Close()
+	if err != nil {
+		t.Error(err)
+		return
+	}
+	sessionID, ticket, sig, _ := defaultWinningTicket(t)
+
+	err = dbh.StoreWinningTicket(sessionID, ticket, sig, nil)
+
+	assert := assert.New(t)
+	assert.NotNil(err)
+	assert.Contains(err.Error(), "nil recipientRand")
+}
+
+func TestLoadtWinningTicket_GivenStoredTicket_LoadsItCorrectly(t *testing.T) {
+	dbh, dbraw, err := TempDB(t)
+	defer dbh.Close()
+	defer dbraw.Close()
+	if err != nil {
+		t.Error(err)
+		return
+	}
+	sessionID, ticket, sig, recipientRand := defaultWinningTicket(t)
+	err = dbh.StoreWinningTicket(sessionID, ticket, sig, recipientRand)
+	if err != nil {
+		t.Fatalf("unexpected errro storing ticket: %v", err)
```

Comment:
Just went ahead and deleted the error messages since the behavior of `require` and `assert` is to print out the failure line, and I think that's good enough for debugging!